PublishedModules/ProjectAlice/ContextSensitive/ContextSensitive.py

```diff
@@ -81,7 +81,7 @@ def addToMessageHistory(self, session: DialogSession):
 
 
 	def lastMessage(self):
-		return self._history[len(self._history) - 1]
+		return self._history[-1]
```

Comment:
doesn't this cause problems when the list is empty on start? (already in the old implementation)

Reply:
returns None now when there is no message in the queue